Dockerfile

```
@@ -0,0 +1,67 @@
+FROM python:3.9-alpine as base
+
+# Set environment variable
+ENV POETRY_HOME="/opt/poetry"
+ENV PATH="$POETRY_HOME/bin:$PATH"
+
+# Install poetry
+RUN pip install poetry Flask gunicorn
```

I'm a fan of handling the Flask/Gunicorn dependencies here exactly as we would other libraries, i.e.:
* Remove them from this statement:
```suggestion
RUN pip install poetry
```
* Make sure both are in your project dependencies (so we'll need to run `poetry add gunicorn` in your host terminal
* Start the app explicitly in the poetry environment, e.g.
`poetry run gunicorn ...` and `poetry run flask ...`